pilot/pkg/model/config.go

```diff
@@ -463,7 +463,7 @@ var (
 	ServiceRole = ProtoSchema{
 		Type:        "service-role",
 		Plural:      "service-roles",
-		Group:       "config",
+		Group:       "rbac",
 		Version:     istioAPIVersion,
```

Comment:
The `istioAPIVersion` is actually `v1alpha2`, you have to change the `ServiceRole`, `ServiceRoleBinding` and `RbacConfig` to use a separate version string if you want to change it to `v1alpha1`.

Reply:
Fixed, Thanks.

Reply:
BTW, I merged you e2e test updates from revision-1.0 to make sure they are passing too.